### PR TITLE
Feat: fullscreen video

### PR DESCRIPTION
**Pull request by sorodrigo**
This PR includes:

- New `<HomeVideo />` component.
- New env var `HOME_VIDEO_ID`.
- Removes annoying webpack dashboard.

How to test:
- copy the `HOME_VIDEO_ID` from `.env.sample` into the `.env` file. Press play button on the home.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -290,6 +290,20 @@ The importer can be started in two ways:
 
 In the first case the importer will run synchronously. In the latter case it will be executed as a background job using sidekiq. For that to work you need redis & sidekiq running. If redis is not already running it can be started using `redis-server`. To start sidekiq in development run `bundle exec sidekiq`. In staging / demo / production starting and stopping sidekiq is handled by capistrano.
 
+### Running the importer in development
```
[the remark this answers is not in the document]
Comment (sorodrigo) [reply README.md]: nope

**Comment by sorodrigo** [comment]
@tiagojsag ready

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tiagojsag) [reply README.md]: @agnessa is this needed?

[the remark this answers is not in the document]
Comment (agnessa) [reply README.md]: not any more - it's no longer in develop so not sure how it made it's way here, maybe some rebasing leftover? Basically this section describes how to set up a tunnel for the db connection which we no longer need to do.

**Pull request merged**